src/libs/actions/Report.js

```diff
@@ -320,6 +320,39 @@ function fetchIOUReportID(debtorEmail) {
     });
 }
 
+function configureReportNameAndIcons(reports, details) {
+    // The personalDetails of the participants contain their avatar images. Here we'll go over each
+    // report and based on the participants we'll link up their avatars to report icons. This will
+    // skip over default rooms which aren't named by participants.
+
+    const reportsToUpdate = {};
+    _.each(reports, (report) => {
+        if (report.participants.length <= 0 && !ReportUtils.isChatRoom(report)) {
+            return;
+        }
+
+        const avatars = ReportUtils.isChatRoom(report) ? (['']) : OptionsListUtils.getReportIcons(report, details);
```

Comment:
It would be nice to have a comment in the code to explain why both of the ternaries exist.